### ISSUE TITLE
Put all strange sentence/section selection articles here

### ISSUE THREAD

**Issue by timvines19** [DataSeer/dataseer-web#313]
- the 'Immunohistochemistry on MSA patient brain' title sentence includes the bioRxiv DOI from two pages before: 
https://app.dataseer.ai/dataseer/documents/608969986a24cc3c8cb3bb51/datasets
https://www.loom.com/share/09b865346beb4836899ec22bed4e632e

**Comment by kermitt2** [comment]
GROBID first is doing a bad job on this document not filtering out entirely these head notes. I added a few bioRxiv examples in GROBID segmentation model training data, but apparently there are still errors and I should add a few more :/  Well, I will add this one for sure :)

For some reasons, GROBID adds then this DOI just before the title in the text body flow (actually it's related to the reading order, the DOI looks on the top of the previous page, but actually it is just before the section title in the PDF streaming order) and, new bug, GROBID also puts it in the same sentence as this section title...  

So to summarize, an very unlucky series of 3 GROBID errors/bugs, each not difficult to fix.

**Comment by timvines19** [comment]
getting the same issue in this one – selecting sentence at line 1122 also bring in DOI at top of previous page

https://app.dataseer.ai/dataseer/documents/6089699f6a24cc3c8cb3bb53/datasets

the sentences lines 1135 to 1144 in this same article are also unselectable.

**Comment by timvines19** [comment]
Sentences are strangely connected in this one – selecting the second half of the first sentence "2A,B), the cells..." on line 971 also selects a sentence on line 942:

https://app.dataseer.ai/dataseer/documents/608969e26a24cc3c8cb3bb61/datasets

https://www.loom.com/share/9e796b1f883b4fe49d024812f0e8e8aa

**Comment by Souad314** [comment]
Other GitHub Open Issues that should probably be linked to here
•      [#40](https://github.com/DataSeer/dataseer-web/issues/40) Text is misplaced into figures 
•	[#46](https://github.com/DataSeer/dataseer-web/issues/46) Data sentences truncate in wrong place
•	[#134](https://github.com/DataSeer/dataseer-web/issues/134) Difficult case for pdf selection
•	[#136](https://github.com/DataSeer/dataseer-web/issues/136) problem processing article
•	[#149](https://github.com/DataSeer/dataseer-web/issues/149) trouble processing F1000 article
•	[#299](https://github.com/DataSeer/dataseer-web/issues/299) Text selecting bug
•	[#301](https://github.com/DataSeer/dataseer-web/issues/301) Sentence bounding rendering issue
•	[#305](https://github.com/DataSeer/dataseer-web/issues/305) Merge selection bug
Also these ML issues because sometimes in these documents where no datasets are selected, it's hard to select the correct sentences/other:
•	[#125](https://github.com/DataSeer/dataseer-web/issues/125) poor performance with uploaded articles
•	[#296](https://github.com/DataSeer/dataseer-web/issues/296) Grobid detection issue

**Comment by Souad314** [comment]
There are three issues with this article, Waschbusch et al, from the ASAP Trial https://app.dataseer.ai/dataseer/documents/6089690e6a24cc3c8cb3bb3d/datasets

I think the : is the reason this selection is off: 
https://www.loom.com/share/668161542d4a41498ed6ccc3daabb3ba

Also, had to manually link these lines even though there's no period after % : 
![image](https://user-images.githubusercontent.com/81999469/120707031-ac3c1f00-c46e-11eb-990b-39c527a6cdd5.png)

Finally (for this doc)
The methods selection was truncated JUST before the required code sentence, so the conflict of interest sentence is selected instead
![image](https://user-images.githubusercontent.com/81999469/120707373-15bc2d80-c46f-11eb-8c20-048df0e7dc49.png)

**Comment by Souad314** [comment]
This AmNat article, [60615_R2.pdf](https://app.dataseer.ai/dataseer/documents/61004f9d86b9b3381ba68de6?documentToken=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJkb2N1bWVudElkIjoiNjEwMDRmOWQ4NmI5YjMzODFiYTY4ZGU2IiwiYWNjb3VudElkIjoiNjA0M2U0MDg3Yzg4NDg0OTZkZGUzNzA4IiwiaWF0IjoxNjI3NDEwMzUzLCJleHAiOjE2MzAwMDIzNTN9.RWBGuUUAJH3sWLjoTUZ0TfVnXarkOZPnaJY3jGll5os)  has entire sections unable to be selected, possibly because of the multicoloured text at the top, but also possibly because of the unique line numbering format.

Here's a loom describing how most things are not selectable
https://www.loom.com/share/807de01d7f9048fd9096d1b1da63c7cb

**Comment by Souad314** [comment]
Now Closed Issues relating to text selection:

- [#496 ](https://github.com/DataSeer/dataseer-web/issues/496) Persisting text selection issues
-[ #461](https://github.com/DataSeer/dataseer-web/issues/461) Sentence Selection issues
- [#441](https://github.com/DataSeer/dataseer-web/issues/441) Strange selection case

- [#400](https://github.com/DataSeer/dataseer-web/issues/400) Most figure legends not selectable text
